contrib/sarif.tpl

```diff
@@ -1,12 +1,14 @@
 {
-  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.4.json",
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
   "version": "2.1.0",
   "runs": [
     {
       "tool": {
         "driver": {
           "name": "Trivy",
+          "informationUri": "https://github.com/aquasecurity/trivy",
           "fullName": "Trivy Vulnerability Scanner",
+          "version": "1.5.0",
```

Comment:
I assume this is the version of the scanning tool, in this case Trivy. So shouldn't this be `v0.15.0`?

Reply:
To be honest I wanted to rely on a template variable at first but there is no metadata available. Only data from vulnerabilities.
So...